### PR TITLE
Patch map in exploring tool

### LINKED ISSUE #3691: Add a patch map view to the auto-evo exploring tool
To let the user see where species are and the world layout

### PR DESCRIPTION
**Pull request by 84634E1A607A**
**Related Issues**

Closes #3691

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [x] Functionality is confirmed working by another person (see above checklist link)
- [x] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

File: src/gui_common/DraggableScrollContainer.cs [hunk 1]
```diff
@@ -82,7 +82,7 @@ public override void _Ready()
     {
         base._Ready();
 
-        ContentPath ??= GetChild(0).GetPath();
+        ContentPath ??= GetChild(2).GetPath();
```
Comment (Maxonovien) [inline src/gui_common/DraggableScrollContainer.cs:85]: Why this change, is it a bugfix ?

Comment (hhyyrylainen) [reply src/gui_common/DraggableScrollContainer.cs:85]: This seems very suspicious to me, and unless the documentation etc. is changed elsewhere this is an incorrect change as this makes the comments and code behaviour different. Also it's very unexpected that this would pick the *third* child as the content if the path is not set.

Comment (Kasterisk) [reply src/gui_common/DraggableScrollContainer.cs:85]: The index definitely shouldn't be any other than 0. ScrollContainers the way it's designed should only have one Control child.

Comment (84634E1A607A) [reply src/gui_common/DraggableScrollContainer.cs:85]: Child 0 is the HScroll bar; 1 VScroll

Comment (Kasterisk) [reply src/gui_common/DraggableScrollContainer.cs:85]: Ah that makes sense, this should be documented then.

**Review by hhyyrylainen — APPROVED** [review]
I had a look through the code and tested in game and I don't really have much to say. I'll open a separate issue about being able to hide the "move to this patch" button in the patch details panel.
Just need to fix the translation conflicts and this can be merged.

**Review by Kasterisk — APPROVED** [review]
